lbrynet/core/Session.py

```diff
@@ -136,6 +137,7 @@ def __init__(self, blob_data_payment_rate, db_dir=None,
         self.payment_rate_manager = None
         self.payment_rate_manager_class = payment_rate_manager_class or NegotiatedPaymentRateManager
         self.is_generous = is_generous
+        self.single_peer_downloader = SinglePeerDownloader()
```

Comment:
why does a session always have a single-peer downloader? this functionality is unnecessary in 99% of cases. it should only be on when we're gonna use it